examples/bjørn_low_weight_fitter_example.py

```diff
@@ -0,0 +1,31 @@
+import sqlite3
```

Comment:
I would rename this file to `examples/bjoern_low_weight_fitter_example.py`, see below.

Reply:
Done